ui_framework/components/panel/_panel.scss

```diff
@@ -1,11 +1,18 @@
 .kuiPanel {
-  border: $globalBorderThick;
+  border: $globalBorderThin;
+  border-radius: $globalBorderRadius;
 }
 
 .kuiPanel--withHeader {
+  // border-top: none;
```

Comment:
Also, the border for the Panel is #E4E4E4 but the border for the ToolBar is #D9D9D9. We should probably use the same value for both.